### ISSUE TITLE
Tell user when CSP fails to load font

### ISSUE THREAD

**Issue by RoelN** [arrowtype/type-x#66]
![image](https://user-images.githubusercontent.com/4570664/68116242-9f55f300-fefa-11e9-85bd-b7e56a190920.png)

See https://github.com/arrowtype/type-x/issues/22#issuecomment-495509886 and https://github.com/arrowtype/type-x/pull/62#issuecomment-549304286

**Comment by RoelN** [comment]
Use something like https://developer.mozilla.org/en-US/docs/Web/API/FontFaceSet and inject a message into the page? Or pass back a flag to the extension and show a warning there and/or in the extension icon? (e.g. add ⚠️)

**Comment by arrowtype** [comment]
One thing that might be worth noting: it (currently) seems to work to reload GitHub when a font has been applied. Perhaps our message to the user could say something like, "Oops, this page doesn't allow automatic font changes. Try refreshing the page to make the font overrides work."

**Comment by arrowtype** [comment]
Of course, that may not be a permanent solution, so I'm not sure how much we should hedge what we tell the user. Maybe something like:

> This page doesn't allow font injection. Reloading the page might help.

**Comment by arrowtype** [comment]
Somehow, https://getsnapfont.com/ is able to override fonts on GitHub and Twitter, which type-x currently can’t. I’m guessing we could figure out how this is possible, and perhaps make type-x that much more useful.

![image](https://user-images.githubusercontent.com/45946693/122651290-d6940a80-d105-11eb-93c1-27baf8a2e084.png)

![image](https://user-images.githubusercontent.com/45946693/122651310-f88d8d00-d105-11eb-8736-9e8355269393.png)

**Comment by RoelN** [comment]
This extension is using fonts served from Google Fonts. I first figured they must've whitelisted GF, but this doesn't seem the case.

Twitter's error when using Type-X is:

`Refused to load the font 'data:application/octet-stream;base64,...' because it violates the following Content Security Policy directive: "font-src 'self' https://*.twimg.com".`

Github's error when using Type-X is:

recursive-code-config:1 Refused to load the font 'data:application/octet-stream;base64,...' because it violates the following Content Security Policy directive: "font-src github.githubassets.com".

These are two different error messages. 

The only difference I see is that our fonts are loaded as base64. I'd need to dig in deeper to find out why our base64 isn't allowed, but external Google Fonts is.
